bigtable/tests/instance_admin_integration_test.cc

```diff
@@ -47,6 +47,18 @@ class InstanceAdminIntegrationTest : public ::testing::Test {
 bool UsingCloudBigtableEmulator() {
   return std::getenv("BIGTABLE_EMULATOR_HOST") != nullptr;
 }
+bool IsInstancePresent(
+    std::vector<::google::bigtable::admin::v2::Instance> instances,
+    std::string instance_name) {
+  bool instance_present = false;
+  for (auto const& i : instances) {
+    if (instance_name.compare(i.name()) == 0) {
+      instance_present = true;
+      std::cout << "Instance Name " << i.name();
+    }
```

Comment:
you can return as soon as the value becomes true, right?  You could also use `std::count()` or `std::find()`, but those are probably harder to read.

Reply:
fixed by breaking the loop on match.